### PR TITLE
feat: ability to set "schemes" programatically

### PR DESCRIPTION
**Pull request by ubogdan**
**Describe the PR**
 Ability to set "schemes" programatically

**Additional context**
PR for ISSUE: #433

### CODE REVIEW

**Comment by ubogdan** [comment]
Readme updated.

File: README.md [hunk 1]
```diff
@@ -194,7 +194,8 @@ func main() {
 	docs.SwaggerInfo.Version = "1.0"
 	docs.SwaggerInfo.Host = "petstore.swagger.io"
 	docs.SwaggerInfo.BasePath = "/v2"
-
+    docs.SwaggerInfo.Schemes = []string{"http", "https"}
```
Comment (easonlin404) [inline README.md]: please align it.

Comment (ubogdan) [reply README.md]: Sorry about alignment are u using it seems that my editor is replacing tabs with different number of spaces.

**Comment by easonlin404** [comment]
@ubogdan Thanks for your contributions. 👍

**Pull request merged**